### ISSUE TITLE
META: wishlist for features and fixes from zcash ecosystem (zcashd, zecwallet-light-cli, etc)

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#204]

**Comment by holmesworcester** [comment]
@NorbertBodziony @jkdev601 are all the items on your mind linked here, currently? Are there any zcashd stability issues we are running into for which we have not created an issue or or found an existing issue? It seems like we're running into quite a bit of instability. 

For example, we should create a ticket for the period of time when zcashd is mysteriously unresponsive in startup. @jkdev601 can you write up this ticket?

We should also create a ticket for zcashd misreporting the total number of blocks in the API call we use to show syncing progress. (Zcashd CLI shows the correct total number of blocks, but the total the RPC interface keeps shifting upward. Can you make a ticket for this one @NorbertBodziony ?

**Comment by NorbertBodziony** [comment]
I think we can drop zcash/lightwalletd#248 it will break ours security model.

**Comment by NorbertBodziony** [comment]
Get tip of blockchain issue zcash/zcash/issues/4591

**Comment by holmesworcester** [comment]
Re: importing viewing keys, it won’t break our security model for username registration channel and “channel of channels” or price oracle since they’re public.

And it would be useful here because if we can query lightwalletd for these things, we can only worry about fetching transactions from *after* the user registers.

**Comment by NorbertBodziony** [comment]
Yes but this is Zbay only use case also we still need ability to read channels that are not public.

**Comment by holmesworcester** [comment]
yeah for that we’ll have to check all transactions.

**Comment by NorbertBodziony** [comment]
Found another bug https://github.com/adityapk00/zecwallet-light-cli/issues/30
